Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1

```diff
@@ -270,13 +271,33 @@ $deploymentTasks = @{
     }
     'Remove-SandboxDatabases'         = {
         $settings = Get-DeploymentSettings
-        if ($settings.ApiSettings.engine -ne 'SQLServer') { return; }
+
+        if ($settings.ApiSettings.engine -ne 'SQLServer' -and $settings.ApiSettings.engine -ne 'PostgreSQL') { return; }
+
```

Comment:
this can be removed now that it works for both engines

Reply:
Done